re.go

```diff
@@ -64,6 +83,12 @@ import (
 // (normally Scan would treat such as a number as octal); or parsing
 // an otherwise unsupported type like time.Duration.
 //
+// func([]byte, Span) error: This is another supported form of custom
```

Comment:
Drop this for now.

Passing &Span suffices for uncommon uses (people can run their custom code later on Span if they want).

Reply:
Done.